tests/e2e/sampleMdFIle.md

```diff
@@ -0,0 +1,14 @@
+## Slide 1
+A paragraph edited finally content one with some text and a [link](https://hakim.se).
```

Comment:
any reason we use the link to this particular company URL in the test content? I think in general we would rather use a non-proprietary URL...